cli/operator/config/create.go

```diff
@@ -49,3 +71,114 @@ func CreateCmd() *cobra.Command {
 
 	return &cmd
 }
+
+func promptOperatorInfo(config *types.OperatorConfig, p prompter.Prompter) (types.OperatorConfig, error) {
+	// Prompt and set operator address
+	operatorAddress, err := p.InputString("Enter your operator address:", "", "",
+		func(s string) error {
+			return validateAddressIsNonZeroAndValid(s)
+		},
+	)
+	if err != nil {
+		return types.OperatorConfig{}, err
+	}
+	config.Operator.Address = operatorAddress
+
+	// Prompt to gate stakers approval
+	gateApproval, err := p.Confirm("Do you want to gate stakers approval?")
+	if err != nil {
+		return types.OperatorConfig{}, err
+	}
+
+	// Prompt for address if operator wants to gate approvals
+	if gateApproval {
+		delegationApprover, err := p.InputString("Enter your staker approver address:", "", "",
+			func(s string) error {
+				return validateAddress(s)
+			},
+		)
+		if err != nil {
+			return types.OperatorConfig{}, err
+		}
+		config.Operator.DelegationApproverAddress = delegationApprover
+	} else {
+		config.Operator.DelegationApproverAddress = zeroAddress
+	}
+
+	// Prompt and set earnings address
+	earningsAddress, err := p.InputString("Enter your earnings address:", config.Operator.Address, "",
+		func(s string) error {
+			return validateAddressIsNonZeroAndValid(s)
+		},
+	)
+	if err != nil {
+		return types.OperatorConfig{}, err
+	}
+	config.Operator.EarningsReceiverAddress = earningsAddress
+
+	// Prompt for eth node
+	rpcUrl, err := p.InputString("Enter your rpc url:", "", "",
+		func(s string) error { return nil },
+	)
+	if err != nil {
+		return types.OperatorConfig{}, err
+	}
+	config.EthRPCUrl = rpcUrl
+
+	// Prompt for ecdsa key path
+	ecdsaKeyPath, err := p.InputString("Enter your ecdsa key path:", "", "",
+		func(s string) error { return nil },
+	)
+	if err != nil {
+		return types.OperatorConfig{}, err
+	}
+	config.PrivateKeyStorePath = ecdsaKeyPath
+
+	// Prompt for bls key path
+	blsKeyPath, err := p.InputString("Enter your bls key path:", "", "",
+		func(s string) error { return nil },
+	)
+	if err != nil {
+		return types.OperatorConfig{}, err
+	}
+	config.BlsPrivateKeyStorePath = blsKeyPath
+
+	// Prompt for network & set chainId
+	chainId, err := p.Select("Select your network:", []string{"mainnet", "goerli", "local"})
+	if err != nil {
+		return types.OperatorConfig{}, err
+	}
+
+	switch chainId {
+	case "mainnet":
+		config.ChainId = *big.NewInt(1)
+	case "goerli":
+		config.ChainId = *big.NewInt(5)
+	case "local":
+		config.ChainId = *big.NewInt(31337)
+	}
+
+	return *config, nil
+}
+
+func validateAddressIsNonZeroAndValid(address string) error {
+	if address == zeroAddress {
+		return errors.New("address is 0")
+	}
+
+	return validateAddress(address)
```

Comment:
maybe use the eigensdk one? - https://github.com/Layr-Labs/eigensdk-go/blob/master/utils/utils.go#L75